lib/widgets/message_list.dart

```diff
@@ -97,6 +97,31 @@ class MessageListAppBarTitle extends StatelessWidget {
   }
 }
 
+class ScrollToBottomButton extends StatelessWidget {
```

Comment:
nit: This should go after the `MessageList` widget, to keep the order consistent with the rest of the file. Note the page widget at the top and the bits around individual messages at the bottom, running from general to specific and root to leaf.